style.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+html body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+.first-section {
+  background: #fff;
+  border-bottom-left-radius: 50px;
+}
+
+.nav-menu ul {
+  display: flex;
+  list-style: none;
+}
+
```

Comment:
- [ ] Please you have extra space at the top of the nav bar so make sure you fix that to match the Figma design :blush: 

![image](https://user-images.githubusercontent.com/92303891/227036956-6ce75ec2-cdb1-4221-b461-b1288585421a.png)